mono/mini/method-to-ir.c

```diff
@@ -11495,7 +11495,6 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 						link_bblock (cfg, tblock, target_bb);
 					}
 				}
-				g_list_free (handlers);
```

Comment:
is it possible to mempool allocate that list?

Reply:
@lewurm why not ?

Reply:
nevermind, I overlooked the change in `mono_find_final_block`